frontend/src/utils/utils.ts

```diff
@@ -62,3 +62,20 @@ export function partition<T>(
     [[], []] as [Array<T>, Array<T>]
   );
 }
+
+/**
+ * @function setDispositionHeader
+ * Constructs a value for http request header 'Content-Disposition'
```

Comment:
Constructs a valid RFC 6266 'Content-Disposition' header and optionally handles RFC 8187 UTF-8 encoding when necessary

Reply:
Added